Plot.R

```diff
@@ -113,241 +137,731 @@ parsesubject <- function(row, half){
-
-window2.end <- as.numeric((chron("12/17/2014", format=c(dates="m/d/y")) - chron("2014-10-21", 
-                                                          format=c(dates="y-m-d"))-1)*24 + 10)
-
-TimeList<- c(seq(window1.begin, window1.end), seq(window2.begin, window2.end))
-TimeList_window1 <- c(seq(window1.begin, window1.end))
-TimeList_window2 <- c(seq(window2.begin, window2.end))
+TimeList_window1 <- c(seq(922, 1232))
+TimeList_window2 <- c(seq(1328, 1545))
 
 ###############################
 # Heatmap of variance explained
 ###############################
 
-transformed.communication.activity <- readr::read_csv("transformed.communication.activity.csv")
+transformed.communication.activity <- 
+  readr::read_csv("transformed.communication.activity.csv")
 transformed.communication.activity["X1"] = NULL
 
 # Two 48 Hour visits containing measurements of blood pressure, heart rate
 BP_HR <- readr::read_csv("heartrate.bp.csv")
-Full <- dplyr::full_join(BP_HR, transformed.communication.activity, by="TimeSubjectIndex")
+Full <- dplyr::full_join(BP_HR, transformed.communication.activity,
+                         by = "TimeSubjectIndex")
 bp.HR.com.act <- na.omit(Full)
 postscript("Variability.Act.Com.BP.eps")
 PlotR2s(bp.HR.com.act[,5:dim(bp.HR.com.act)[2]], 
-        "Variance Explained in Activity, Communication, Biometric Data (Visits 1 and 2)", "BP")
+        paste("Variance Explained in Activity,", 
+              " Communication, Biometric Data (Visits 1 and 2)", "BP"))
+
 dev.off()
 
+
 # 48 hour visit 1
 Visit1_bp.HR.com.act <- subset(bp.HR.com.act, Times %in% TimeList_window1)
 
-postscript("Varability.Act.Com.BP.Visit1")
+postscript("Varability.Act.Com.BP.Visit1.Feb.eps")
 PlotR2s(Visit1_bp.HR.com.act[, 5:dim(Visit1_bp.HR.com.act)[2]],
-        "Variance Explained in Activity, Communication, Biometric Data(Visit 1)", "BP")
+        "Variance Explained in Activity, Communication, Biometric Data(Visit 1)",
+        "BP")
 dev.off()
 
 # 48 hour visit 2
 Visit2_bp.HR.com.act <- subset(bp.HR.com.act, Times %in% TimeList_window2)
-postscript("Varability.Act.Com.BP.Visit2")
+postscript("Varability.Act.Com.BP.Visit2.Feb.eps")
 PlotR2s(Visit2_bp.HR.com.act[, 5:dim(Visit2_bp.HR.com.act)[2]],
         "Variance Explained in Activity, Communication, Biometric Data (Visit 2)", "BP")
 dev.off()
 
+energy <- read.csv("energy.csv")
+energy["X"] <- NULL
+energy["Times"] <- apply(energy, 1, parsesubject, 1)
+
+Energy.4months <- dplyr::full_join(transformed.communication.activity, energy,
+                                   by = "TimeSubjectIndex")
+Energy.4months <- na.omit(Energy.4months)
+
+Full.with.energy <- dplyr::full_join(bp.HR.com.act, energy,
+                                     by = "TimeSubjectIndex")
+Full.with.energy <- na.omit(Full.with.energy)
+Full.with.energy["Subject"] <- NULL
+
+postscript("Variance_Explained_WithEnergy.eps")
+PlotR2s(Full.with.energy[, 6:dim(Full.with.energy)[2] - 1],
```

Comment:
Can you put a little more info here about what's happening with the column index?